tests/robustness/watch.go

```diff
@@ -112,7 +115,7 @@ func watchMember(ctx context.Context, t *testing.T, c *clientv3.Client, maxRevis
 			if resp.Err() == nil {
 				// using time.Since time-measuring operation to get monotonic clock reading
```

Comment:
nit: move this comment into `traffic.WatchResponse` or remove it

Reply:
Done